scripts/partitioned/migration/1-create-tables.sql

```diff
@@ -0,0 +1,40 @@
+DROP TABLE IF EXISTS public.journal_partitioned CASCADE;
+CREATE TABLE IF NOT EXISTS public.journal_partitioned
+(
+    ordering        BIGSERIAL,
+    persistence_id  TEXT                  NOT NULL,
+    sequence_number BIGINT                NOT NULL,
+    deleted         BOOLEAN DEFAULT FALSE NOT NULL,
+    tags            int[],
+    message         BYTEA                 NOT NULL,
+    PRIMARY KEY (persistence_id, sequence_number)
+) PARTITION BY LIST (persistence_id);
+
+CREATE INDEX journal_partitioned_ordering_idx ON public.journal_partitioned USING BRIN (ordering);
```

Comment:
Do not forget about GIN index for tags column

Reply:
hmm.. I've coppied it from `partitioned-schema.sql` .

Reply:
I now see that you add it on your branch